README.md

```diff
@@ -3,3 +3,127 @@
 [Hydra](http://www.hydra-cg.com/spec/latest/core/) is a machine readable description for APIs.
 Hydra Box extends the API description with links to the actual code, which provides the API.
 Hydra Box will use such an API description to start a server which provides the API and dynamically loads the required code for it.
+
+## Getting started
+
+A hydra-box API is essentially an [expressjs](https://expressjs.com) app where it is yet another middleware. Three main steps 
```

Comment:
Is it an express app or an express middleware?

Reply:
hydra-box is a middleware. But the API is an express app. Ideas for improvement welcome

Reply:
At the moment it's just a middleware, but I think we can/should add a generic server, configurable via cli args.

Reply:
Ok, let me reword that